### PR TITLE
Poke Response Should Show Success/Failure instead of JSON

### PR DESCRIPTION
**Pull request by hellowhirl**

### CODE REVIEW

File: visor-extension/src/components/modal/Display.tsx [hunk 1]
```diff
@@ -41,15 +45,20 @@ const Display = (props: DisplayProps) => {
...
-            onClick={event => navigator.clipboard.writeText(JSON.stringify(props.airlockResponse))}
-          >
-            copy
-          </button>
+          <div className="command-launcher-display-preview-container">
+            <div className="command-preview-icon">
+              {response === '"Poke Successful"' || response === '"Poke Failed"' ? (
+                <PokeIcon />
+              ) : (
+                <ThreadIcon />
```
Comment (h5gq3) [inline visor-extension/src/components/modal/Display.tsx:57]: imo would be better to not render icon in case the response wasn't poke success/error

Comment (hellowhirl) [reply visor-extension/src/components/modal/Display.tsx:57]: That's a good point. The response could be something else besides a Poke or Thread command, right? Like what else though..?

Comment (h5gq3) [reply visor-extension/src/components/modal/Display.tsx:57]: for responses, i'd just render the response string alone without icon. not sure if in figma there was with icon or not. i still don't have access to the figma file...

**Comment by hellowhirl** [comment]
@robkorn conflict resolved - please review

**Pull request merged**